release/Magarena/scripts/Tezzeret_s_Simulacrum.groovy

```diff
@@ -21,11 +21,7 @@
         public void executeEvent(final MagicGame game, final MagicEvent event) {
             event.processTargetPlayer(game, {
                 final MagicPlayer target ->
-                final int amount =
-                    (event.getPlayer().getPermanents().any({
-                        it.hasType(MagicType.Planeswalker) &&
-                        it.hasSubType(MagicSubType.Tezzeret)
-                    })) ? 3 : 1;
+                final int amount = (event.getPlayer().controlsPermanent(MagicType.Artifact)) ? 3 : 1;
```

Comment:
This also changes the behavior of the card, the original code is correct in that it checks for Tezzeret planeswalker. The new code checks if player controls an Artifact. You can use `event.getPlayer().controlsPermanent(planeswalker(MagicSubType.Tezzeret, Control.You))`

Reply:
Right. Sorry I somehow misremembered the condition.